pkg/kv/kvserver/allocator_scorer.go

```diff
@@ -2082,11 +2064,6 @@ func (o storeHealthOptions) rebalanceToReadAmpIsHealthy(
 	return o.enforcementLevel < storeHealthBlockRebalanceTo
 }
 
-// maxCapacityCheck returns true if the store has room for a new replica.
-func maxCapacityCheck(store roachpb.StoreDescriptor) bool {
-	return store.Capacity.FractionUsed() < maxFractionUsedThreshold
-}
-
 // rebalanceToMaxCapacityCheck returns true if the store has enough room to
```

Comment:
Should this also go along to base.go, with the other disk fullness check? I'm don't think there's as much benefit from splitting this off from the scorer yet.

Reply:
See comment above.